### PR TITLE
I18n

### PR DESCRIPTION
**Pull request by giorgio93p**
The module has some messages in html/js, which will be internationalized in the future (when workbench supports it)

Looks like the number 10 in `TooManyTextValuesError.message` must be replaced with a parameter taking the value of `MaxNAxisLabels`. I am not 100% sure, though.

### CODE REVIEW

File: linechart.py [hunk 1]
```diff
@@ -72,6 +73,132 @@ class GentleValueError(ValueError):
     """
 
 
+class RenderingError(ValueError):
```
Comment (adamhooper) [inline linechart.py]: Nix this exception hierarchy, please. Raise `GentleValueError(i18n.trans(...))` instead.

The error handler should look to `err.args[0]`.

Unit tests should test the I18nMessage.

File: linechart.py [hunk 2]
```diff
@@ -389,8 +495,8 @@ def render(table, params, *, input_columns):
         chart = form.make_chart(table, input_columns)
     except GentleValueError as err:
         return (table, "", {"error": str(err)})
-    except ValueError as err:
-        return (table, str(err), {"error": str(err)})
+    except RenderingError as err:
```
Comment (adamhooper) [inline linechart.py]: Unfortunately, `str(err)` on the line above won't be translatable for the forseeable future. So let's nix it.

Pass `{"error": error: err.args[0]}` -- even though we won't be translating it. Modify `linechart.html`'s `errorSpec()` to set `"title": "Please correct the error in this step's data or parameters"`. In the future, we'll probably have a good way to handle this data.

Comment (giorgio93p) [reply linechart.py]: `err.args[0]` is not JSON serializable. I am passing `"Please correct the error in this step's data or parameters"` for now

File: tests/test_linechart.py [hunk 3]
```diff
@@ -136,35 +131,94 @@ def build_form(self, **kwargs):
 
     def test_missing_x_param(self):
         form = self.build_form(x_column="")
-        with self.assertRaisesRegex(GentleValueError, "Please choose an X-axis column"):
+        with self.assertRaises(GentleValueError):
```
Comment (adamhooper) [inline tests/test_linechart.py]: No need to call this twice.

```
with self.assertRaises(GentleValueError) as cm:
    # code that raises

self.assertEqual(cm.exception.i18n_message, ...)
```

**Pull request merged**